src/components/checkboxWithLabel.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+type props = {
+    labelOn: string,
+    labelOff: string
+}
+export class CheckboxWithLabel extends React.Component<{
```

Comment:
Why we have used class components 
change the into functional

Reply:
Changed at every instance.

Reply:
Removed the component (it was not used).